pandas/core/generic.py

```diff
@@ -1408,16 +1408,18 @@ def __neg__(self):
 
     def __pos__(self):
         values = self._values
-        if is_bool_dtype(values):
-            arr = values
-        elif (
-            is_numeric_dtype(values)
+        if (
+            is_bool_dtype(values)
+            or is_numeric_dtype(values)
             or is_timedelta64_dtype(values)
             or is_object_dtype(values)
         ):
-            arr = operator.pos(values)
+            arr = values
```

Comment:
why did you remove this? we need to dispatch to the object itself in case it needs to actually take action

Reply:
This was left over from before I added `__pos__`, will fix now